test/amoc_api_status_handler_SUITE.erl

```diff
@@ -4,260 +4,76 @@
 -include_lib("eunit/include/eunit.hrl").
 -export([all/0, init_per_testcase/2, end_per_testcase/2]).
 
--export([test_success/1,
-         test_fail/1,
-         test_list/1,
-         test_start_fail_no_scenario/1,
-         test_start_success/1,
-         test_load_good/1,
-         test_load_compilation_err/1,
-         test_ping_nodes/1,
-         test_get_test_result/1,
-         test_get_tests_results/1,
-         test_get_wrong_test_result/1]).
+-export([works_when_amoc_running_offline/1,
```

Comment:
The test cases' names are not too good. Please refer to our rules to make them better.

For example: `get_status_returns_false_when_amoc_down`.